app/src/NativeBase/Components/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file CheckBox component for the STA Volunteer App
+ */
+
+import { Pressable } from 'native-base'
+import React from 'react'
+import { Ionicons } from '@expo/vector-icons'
+
+type CheckboxProps = {
+  onChange: (value: string[]) => void
+  checked: boolean
+}
+
+/**
+ * Component that displays a checkbox that can be checked or unchecked.
+ *
+ * @param { CheckboxProps} props The component props
+ * @param {value: string[] => void} props.onChange function called when the checkbox is pressed
+ * @param {boolean} props.checked whether the checkbox is checked or not
+ * @returns {React.ReactElement} Component
+ */
+
+const Checkbox: React.FC<CheckboxProps> = ({ onChange, checked }) => {
+  return (
+    <Pressable
+      onPress={() => onChange([])}
```

Comment:
Why does `onChange` take a `value` parameter if it doesn't do anything with it?  If it's not actually being used, could we just remove this and then this line could simply be `onPress={onChange}`?

(I was also confused as to why a single checkbox would return an array of strings, not just a string)